src/ClientData/include/ClientData/ThreadStateSliceInfo.h

```diff
@@ -62,6 +69,7 @@ class ThreadStateSliceInfo {
   WakeupReason wakeup_reason_;
   uint32_t wakeup_tid_;
   uint32_t wakeup_pid_;
+  std::optional<uint64_t> triggering_callstack_id_;
```

Comment:
I would prefer the name `switch_out_or_wakeup_callstack_id` that we use in the protos.

Reply:
Same in all the other places in this change.

Reply:
Done.